exec/java-exec/src/main/java/org/apache/drill/exec/server/options/OptionValidator.java

```diff
@@ -69,6 +75,13 @@ public int getTtl() {
   }
 
   /**
+   * @return true is option can be set on system level only
```

Comment:
Explain a bit better. What is meant by "on system level only"? That this can be set only for the whole Drill system? That this is a system-level property that only the admin (not user) can set?